BE/FunctionalEntities/FunctionalEntities.rdf

```diff
@@ -132,21 +111,113 @@
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label>merchant</rdfs:label>
-		<skos:definition>a business entity engaged in a trading activity</skos:definition>
+		<skos:definition>party engaged in the purchase and sales of goods for profit</skos:definition>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-be-fct-fct;MerchantCategoryCode">
+		<rdfs:subClassOf rdf:resource="https://spec.edmcouncil.org/fibo/ontology/FND/Arrangements/ClassificationSchemes/IndustrySectorClassifier"/>
+		<rdfs:subClassOf rdf:resource="&lcc-lr;CodeElement"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;isDefinedIn"/>
+				<owl:onClass rdf:resource="&fibo-be-fct-fct;MerchantCategoryCodeScheme"/>
+				<owl:qualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">1</owl:qualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-be-fct-fct;hasGeneralMerchantCategoryDescription"/>
+				<owl:onDataRange rdf:resource="&xsd;string"/>
+				<owl:qualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">1</owl:qualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-be-fct-fct;hasSpecificMerchantCategoryDescription"/>
+				<owl:onDataRange rdf:resource="&xsd;string"/>
+				<owl:qualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">1</owl:qualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&lcc-lr;hasTag"/>
+				<owl:onDataRange rdf:resource="&xsd;string"/>
+				<owl:qualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">1</owl:qualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&lcc-cr;classifies"/>
+				<owl:someValuesFrom rdf:resource="&fibo-be-fct-fct;Merchant"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label>merchant category code</rdfs:label>
+		<skos:definition>code used internationally to classify a merchant</skos:definition>
+		<fibo-fnd-utl-av:abbreviation>MCC</fibo-fnd-utl-av:abbreviation>
+		<fibo-fnd-utl-av:adaptedFrom>ISO 18245:2003 Retail financial services - Merchant category codes</fibo-fnd-utl-av:adaptedFrom>
+		<fibo-fnd-utl-av:explanatoryNote>Merchant categories are organized by the type of business, trade or services supplied.  Certain category codes including those for very large businesses, such as airlines and some hotel chains, may be delineated to the point of identifying the business.</fibo-fnd-utl-av:explanatoryNote>
+		<fibo-fnd-utl-av:explanatoryNote>Merchant category codes and/or the descriptions of the service categories are frequently used in credit card and other banking transactions for analysis and sometimes tax-related purposes.</fibo-fnd-utl-av:explanatoryNote>
```

Comment:
More importantly they're used for rewards. And advertised as part of the credit card product (e.g. you get 4% cash back on fuel purchases)